### PR TITLE
published accepted proposal and updated mailing lists

### PR DESCRIPTION
**Pull request by aasmall**
we received permission from the vendor to share the proposal we accepted. I've added it, as well as updated the mailing lists.

### CODE REVIEW

**Comment by joelsmith** [comment]
/lgtm
Sorry I didn't see this earlier and now it needs a rebase. ☹️  I'll give another lgtm post-rebase, but I'm probably not an approver for enough of the tree for my lgtm to matter for the entire PR.  I think somebody high enough should give lgtm too. Thanks!

**Comment by cblecker** [comment]
I've approved it already, so you should be able to lgtm it again after rebase, @joelsmith

File: wg-security-audit/README.md [hunk 1]
```diff
@@ -21,14 +21,19 @@ Perform a security audit on k8s with a vendor and produce as artifacts a threat
...
 The RFP will be open between 2018/10/29 and 2018/11/30 and has been published [here](https://github.com/kubernetes/community/blob/master/wg-security-audit/RFP.md).
 
-## Submission
+## Vendor Selection
+
+The [RFP](https://github.com/kubernetes/community/blob/master/wg-security-audit/RFP.md) is now closed. The working group selected Trail of Atredis, a collaboration between [Trail of Bits](https://www.trailofbits.com/) and [Atredis Partners](https://www.atredis.com/) to perform the audit.
+
+## Mailing Lists
+
+* Sensitive communications regarding the audit shouls be sent to the [private variant of the mailing list](https://groups.google.com/forum/#!forum/kubernetes-wg-security-audit-private).
```
Comment (justaugustus) [inline wg-security-audit/README.md]:
```suggestion
* Sensitive communications regarding the audit should be sent to the [private variant of the mailing list](https://groups.google.com/forum/#!forum/kubernetes-wg-security-audit-private).
```

**Pull request merged**